packages/vscale/src/log-scale.ts

```diff
@@ -147,7 +150,7 @@ export class LogScale extends ContinuousScale {
    */
   forceTicks(count: number = 10): any[] {
     const d = this.calculateVisibleDomain(this._range);
-    return forceTicks(d[0], d[d.length - 1], count);
+    return forceTicksBaseTransform(d[0], d[1], count, this.transformer, this.untransformer);
```

Comment:
don't read `d[1]`, just use `d[d.length - 1]`, in same case ,domain has length > 2